internal/activities/validate_structure.go

```diff
@@ -74,6 +74,22 @@ func (a *ValidateStructure) Execute(
 		failures = append(failures, extras...)
 	}
 
+	// Check that digitized SIPs only have one dossier in the content dir.
+	if params.SIP.Type == enums.SIPTypeDigitizedSIP {
+		entries, err := os.ReadDir(params.SIP.ContentPath)
+		if err != nil {
+			return nil, fmt.Errorf("ValidateStructure: check for unexpected dossiers: %v", err)
+		}
+
+		if len(entries) > 1 {
+			for _, entry := range entries[1:] {
+				failures = append(failures, fmt.Sprintf(
+					"Unexpected dossier: %s", entry.Name(),
+				))
```

Comment:
This could go in one line. If not, I prefer to split each argument on its own line:

```go
failures = append(
    failures,
    fmt.Sprintf("Unexpected dossier: %s", entry.Name()),
)
```